rpcserver.go

```diff
@@ -3857,11 +3856,23 @@ func (r *rpcServer) fetchWaitingCloseChannels() (waitingCloseChannels,
 			Memo:                  string(waitingClose.Memo),
 		}
 
+		var closingTxid string
+		var closingTxHex string
+		if closingTx != nil {
+			var txBuf bytes.Buffer
+			if err := closingTx.Serialize(&txBuf); err != nil {
+				return nil, 0, err
+			}
+			closingTxHex = hex.EncodeToString(txBuf.Bytes())
+			closingTxid = closingTx.TxHash().String()
+		}
+
 		waitingCloseResp := &lnrpc.PendingChannelsResponse_WaitingCloseChannel{
 			Channel:      channel,
 			LimboBalance: channel.LocalBalance,
 			Commitments:  &commitments,
 			ClosingTxid:  closingTxid,
+			ClosingTxHex: closingTxHex,
```

Comment:
Let's make this an optional field and only populate it when specified via `PendingChannelsRequest`?

Reply:
Makes sense, done.